src/main/resources/blockly/js/mcreator_blockly.js

```diff
@@ -90,6 +90,48 @@ function jsonToBlocklyDropDownArray(json) {
     return retval;
 }
 
+// Helper function to use in Blockly extensions that check if output types of some children blocks are the same
+// If sourceInput name is provided, types of inputs are compared to input with that name
+// Otherwise, if the block is connected to a value input, types of inputs are compared to types accepted by that input
+function validateInputTypes(inputNames, sourceInput, repeatingInputs=false) {

```

Comment:
Should this function accept both list of regular input names and list of repeating input names at the same time?